content/en/docs/Development/creating_derivatives.md

```diff
@@ -143,17 +92,7 @@ requires:
   version: ">=0"
 ```
 
-The difference is that with `requires` we use the _building_ container that was used to build the packages instead of creating a new image from their results: we are not consuming their artifacts in this case, but the environment used to build them. See also [the luet docs](https://luet-lab.github.io/docs/docs/concepts/packages/specfile/#package-source-image) for more details. 
-
-### Building
-
-Refering to the `sampleOS` example, we set the [Makefile](https://github.com/rancher-sandbox/cos-toolkit-sample-repo/blob/8ed369c6ca76f1fc69e49d8001c689c8d0371d30/Makefile#L13) accordingly to compile the system package.
-
-With luet installed locally and docker running, in your git checkout you can build it also by running `luet build --tree packages system/sampleOS`. This will produce an artifact of `system/sampleOS`. Similary, we could also build separately the sample application with `luet build --tree packages app/sampleOSService`.
-
-The build process by default results in a `build` folder containing the package and the compilation metadata in order to generate a repository.
-
-_Note on reproducibility_: See [the difference between our two samples repositories](https://github.com/rancher-sandbox/epinio-appliance-demo-sample#main-difference-with-cos-toolkit-sample-repo) for an explanation of what are the implications of using a `.luet.yaml` file for building instead of a git submodule.
+The difference is that with `requires` we use the _building_ container that was used to build the packages instead of creating a new image from their results: we are not consuming their artifacts in this case, but the environment used to build them. See also [the luet docs](https://luet-lab.github.io/docs/docs/concepts/packages/specfile/#package-source-image) for more details.
 
 ## Additional packages
```

Comment:
This section below should be removed as well, It's still refering to the sample repositories